pkgs/applications/editors/kakoune/plugins/kak-idris.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv, fetchFromGitHub, idris2, nodejs }:
+stdenv.mkDerivation {
+  name = "kak-idris";
+  version = "2020-12-29";
+
+  src = fetchFromGitHub {
+    owner = "stoand";
+    repo = "kakoune-idris";
+    rev = "1acdfb5d89e3951ae4bdf4a5fa2377b36448083d";
+    sha256 = "06qny8790j9d7vjs0pfyw71xvc6hmkjl8ssi483mbwbxs0zv6j9r";
+  };
+
+  patchPhase = ''
+    sed -i 's \bnode\b ${nodejs}/bin/node g' idris.kak
```

Comment:
apparently `|` is preferred as the alternative sed-separator if `/` is unworkable...